.github/workflows/build-and-test.yml

```diff
@@ -52,11 +52,13 @@ jobs:
 
       - name: Unit and integration tests for all crates in workspace
         run: |
+          export AZTEC_SRS_PATH="$PWD/data/aztec20/kzg10-aztec20-srs-1048584.bin"
+          ./scripts/download_srs_aztec.sh
```

Comment:
I wonder if this can be a justfile command or something.

Reply:
okay! i will do that, I like single responsibility principle.

Reply:
I'll re-approve when you add it then!

Reply:
done in [c3fd331](https://github.com/EspressoSystems/HotShot/pull/2854/commits/c3fd33151aecb47f6abaa1df0fa5b3c11052a6fe)

Reply:
ah! the env var `AZTEC_SRS_PATH` is needed in the subsequent steps, and setting env var in just command is not possible (afaik), it's only good at reading it or temporarily setting it within the scope of a just command ([see here](https://just.systems/man/en/chapter_37.html))

so I will revert back. lmk if you disagree @jparr721

Reply:
I think that that is fine. Using a `scripts/` folder is a perfectly idiomatic thing as well.